### PR TITLE
Feat: Optimizely custom metrics

### PR DESCRIPTION
**Pull request by christian14b**
This pr aims to track custom events:
- Deposits
- Loan share purchase
using [Optimizely api](https://docs.developers.optimizely.com/web/docs/event).

### CODE REVIEW

File: src/pages/Thanks/ThanksPage.vue [hunk 1]
```diff
@@ -352,6 +358,17 @@ export default {
 				);
 			}
 		}
+
+		if (this.$appConfig.enableOptimizely) {
```
Comment (mcstover) [inline src/pages/Thanks/ThanksPage.vue:362]: Generally we're tracking these type of transaction events here: https://github.com/kiva/ui/blob/main/src/plugins/kv-analytics-plugin.js#L214

Thus, you should be able to skip any changes to the Thanks page, import your new event util (or just embed that one method) and use the transactionData we've already aggregated for these events.

Comment (emuvente) [reply src/pages/Thanks/ThanksPage.vue:362]: +1

**Review by mcstover — APPROVED** [review]

File: src/plugins/kv-analytics-plugin.js [hunk 2]
```diff
@@ -331,6 +337,25 @@ export default {
 					event: 'refresh'
 				});
 			},
+			trackOPTransaction: transactionData => {
+				window.optimizely.push({
```
Comment (emuvente) [inline src/plugins/kv-analytics-plugin.js:341]: I think these events should only fire when the deposit total/loan total is greater than 0, and the only tag that's needed would be the deposit amount/loan share purchase amount.

**Review by emuvente — APPROVED** [review]
👍 Looks good!

**Pull request merged**